tests/cypress/fixtures/gr-ni-interface-output.txt

```diff
@@ -0,0 +1,3 @@
+Strain matrix as percentages:
```

Comment:
Why do we need a file for this?

Reply:
Previously there was a larger output so it was reasonable to hide in a file. I can switch step to use docstring.